development.json

```diff
@@ -1,5 +1,5 @@
 {
   "require": {
-    "wpackagist-plugin/elasticpress":"4.7.*"
+    "wpackagist-plugin/elasticpress":"5.1.*"
```

Comment:
We can completely remove `development.json` as it's identical to the base composer file